app/controllers/users_controller.rb

```diff
@@ -1,16 +1,30 @@
 class UsersController < ApplicationController
-
   def new
     @user = User.new
   end
 
   def create
-   @user = User.new(user_params)
+    @user = User.new(user_params)
+    if @user.save
```

Comment:
New line between 7-8?

Reply:
This is fixed with the change in 1fb4972